libglusterfs/src/event-epoll.c

```diff
@@ -36,23 +35,33 @@ struct event_slot_epoll {
     gf_lock_t lock;
 };
 
+struct event_slot_epoll_table {
+    int table_slots_used;
```

Comment:
Probably better to just use `slots_used`.
```suggestion
    int slots_used;
```

Reply:
Done. It was easier for me when I edited the code, but now I've reverted back to the old name.